libqtile/widget/net.py

```diff
@@ -29,43 +29,45 @@ class Net(base.ThreadedPollText):
     """Displays interface down and up speed"""
     orientations = base.ORIENTATION_HORIZONTAL
     defaults = [
-        ('interface', 'wlan0', 'The interface to monitor'),
+        ('interface', None, 'List of interfaces to monitor or None to displays all active NICs combined'),
```

Comment:
You should leave in the doc that a string will be interpreted as a single interface.

Reply:
The doc should be fine now
```python
defaults = [
        ('interface', None, 'List of interfaces or single NIC as string to monitor, None to displays all active NICs combined'),
        ('update_interval', 1, 'The update interval.'),
    ]
```